cla_backend/settings/base.py

```diff
@@ -266,9 +266,15 @@
         'console': {
             'level': 'INFO',
             'class': 'logging.StreamHandler',
-            'formatter': 'logstash',
+            'formatter': 'simple',
```

Comment:
🙋‍♂️ It would be great to find a way to avoid nested JSON but still have a JSON outcome. Is that what will happen?

Reply:
Added sample to JIRA 🎟